packages/page-staking/src/Performance/CurrentList.tsx

```diff
@@ -13,34 +13,46 @@ import Address from './Address';
 
 interface Props {
   className?: string;
-  toggleFavorite: (address: string) => void;
   session: number;
-  validatorPerformances: ValidatorPerformance[];
+  eraValidatorPerformances: EraValidatorPerformance[];
 }
 
-function sortValidatorsByFavourites (validatorPerformances: ValidatorPerformance[]): ValidatorPerformance[] {
-  return validatorPerformances
-    .sort(({ isFavourite: favA }: ValidatorPerformance, { isFavourite: favB }: ValidatorPerformance): number => {
-      return favA === favB ? 0 : (favA ? -1 : 1);
-    });
+function getFiltered (displayOnlyCommittee: boolean, eraValidatorPerformances: EraValidatorPerformance[]) {
+  const validators = displayOnlyCommittee ? eraValidatorPerformances.filter((performance) => performance.isCommittee) : eraValidatorPerformances;
+
+  return validators;
 }
 
-function getFiltered (displayOnlyCommittee: boolean, validatorPerformances: ValidatorPerformance[]) {
-  const validators = displayOnlyCommittee ? validatorPerformances.filter((performance) => performance.isCommittee) : validatorPerformances;
+export function calculatePercentReward (blocksCreated: number | undefined, blocksTargetValue: number) {
+  if (blocksCreated === undefined) {
+    return '';
+  }
+
+  let rewardPercentage = 0;
+
+  if (blocksTargetValue > 0) {
+    rewardPercentage = 100 * blocksCreated / blocksTargetValue;
+
+    if (rewardPercentage >= 90) {
```

Comment:
actually this shiuld be `100 >= rewardPercentage >= 90`

Reply:
Good catch - will fix